paper.md

```diff
@@ -148,68 +148,69 @@ While the main text of this paper remains fairly theoretical, we aim to lay out
+
+We have experimented with numerous designs in delivering our hack week tutorials in a way that is student-centered and focused on meeting each student at their particular skill level. We have observed that, especially during long tutorials and with a particularly diverse group of students, there is a natural separation between participants who are focused on gaining new knowledge as it is being presented, versus those who already possess this knowledge and hence turn their attention to other subjects as the tutorial is ongoing. In practice, this concern can be mitigated by explicitly involving experienced participants in the tutorials. We encourage instructors to call upon experts in the audience to identify themselves and effectively act as teaching assistants. Expert participants often find this gratifying because it allows them to test their own understand and improve their skills by learning through teaching. At the same time, beginners benefit by having increased interactions with other hack week participants, thereby increasing team cohesion and building community. Another technique to engage expert participants during the tutorial is to invite them to work through more challenging content located in the online lesson, which will not be covered in the session, but for which they can obtain help during the tutorial.
+
+Many of our hack week tutorials follow the structure of the Software Carpentry model **[I'm not sure if that is true for all?]**. The content is divided in to a series of lessons, each having a well-defined set of learning objectives, key questions, and expected outcomes. Interspersed throughout each lesson are a series of exercises which participants work through at their own pace. We frequently use a method drawn from Software Carpentry whereby students indicate their need for help on an exercise via posting a colored sticky-note on their computer. This provides a simple method to deliver help to those students who need it without interrupting the flow of the tutorial. We encourage instructors to develop online material to accompany their tutorials, either in the form of Jupyter Notebooks or following the markdown templates provided by Software Carpentry. This way the hack week "curriculum" can be built up over time and students can refer to online content after the tutorial if they need to revisit certain topics. Depending on the content, some instructors convey concepts during the tutorial via "live-coding" while others display static code and their results. Either way, we find diving code into short blocks, each with a specific task, helps participants from becoming saturated with informaiton.
```

Comment:
Astro Hack Week hasn't really relied that explicitly on the Software Carpentry model, but reading this, we probably should. 

Last sentence: `[...] we find diving code into short blocks` == `dividing`?